Kong/docs.konghq.com
# Third party plugins cleanup: Bitnami Azure deployment

### Description

Removing the Kong on Azure by Bitnami deployment guide as part of the third-party plugin cleanup initiative.
Reasons:
* not a plugin
* not a kong-owned deployment option
* we have our own packages on the microsoft marketplace:
  * Bitnami listing: https://azuremarketplace.microsoft.com/en-us/marketplace/apps/bitnami.kong
  * Kong-owned listings: https://azuremarketplace.microsoft.com/en-us/marketplace/apps/konginc1581527938760.kongee2020ubuntuxenial and https://azuremarketplace.microsoft.com/en-us/marketplace/apps/konginc1581527938760.kong-enterprise

### Testing instructions

Netlify link: https://deploy-preview-5496--kongdocs.netlify.app/hub/bitnami/ should redirect to /gateway/latest/install/


### Checklist
